docs/workforce-management-framework/topics/pro-working-with-passportjs.adoc

```diff
@@ -0,0 +1,41 @@
+[id='pro-working-with-passportjs-{chapter}']
+= Introduction
+
+This section includes:
+
+. xref:pro-configuring-passportjs-{chapter}[Configuring Passport.js]
+. xref:pro-implementing-the-userrepository-{chapter}[Implementing the UserRepository]
+. xref:pro-modifying-a-strategy-{chapter}[Modifying a Strategy]
+
+= Prerequisites
+(?)
+
+= Procedure
+
+[id='pro-configuring-passportjs-{chapter}']
```

Comment:
Can we turn the content in the 3 headings into something that resembles the delivery of actions in "docs/workforce-management-framework/topics/pro-running-the-demo-app.adoc"?

Note: this adoc is one that I didn't get to review in full as time is something I'm short on atm hence the "90% docs approved" statement during our call

Reply:
not sure what you mean , could hammer it out on a call.